src/Compilers/Core/CodeAnalysisTest/Text/TextChangeTests.cs

```diff
@@ -739,9 +739,10 @@ public void TestMergeChanges_BeforeAdjacent()
             Assert.Equal("o World", changes[0].NewText);
         }
 
-        [Fact]
+        [ConditionalFact(typeof(ClrOnly), Reason = "https://github.com/mono/mono/issues/10961")]
         public void TestMergeChanges_NoMiddleMan()
         {
+            try {
```

Comment:
>try { [](start = 12, length = 5)

nit: format of the try/catch and catch block

Reply:
Sorry, missed that. deleting.